connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/inbound/InboundConnectorContextImpl.java

```diff
@@ -155,6 +151,11 @@ public void reportHealth(Health health) {
     this.health = health;
   }
 
+  @Override
+  public void loadCancellationCallback(Consumer<Throwable> cancellationCallback) {
```

Comment:
I would call it `set` not `load` as it's basically a setter

Reply:
Correct